core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -380,18 +380,17 @@ export type RealityModelSource = ViewState | DisplayStyleState;
 
 /** @internal */
 export class RealityModelTileTree extends RealityTileTree {
+  public readonly _isContentUnbounded: boolean;

```

Comment:
If this is `public` it shouldn't have the leading underscore (linter will complain I think).

Reply:
Yes... intended to make that private.   Fixed.